### DISCUSSION TITLE
How do I change the extension language?

### DISCUSSION THREAD

**Discussion by faq0** [BewlyBewly/BewlyBewly#619] [Q&A] [answered]
My system language is English, and therefore the extention language is set to English. Is there any option, in the extension, to change the language to Chinese (Simplified) or Chinese (Traditional)?

Thanks.

**Comment by hakadao** [comment, marked as answer]
You can open the settings, which are located in the dock.
Then in the "General" settings, the first option is the one you need to change the display language

**Comment by faq0** [comment]
Thanks. I didn't notice that earlier.
